src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -33,11 +34,25 @@ class MemberPermission(OrganizationPermission):
     }
 
 
+SORT_MAP = {
+    "popularity": "popularity",
+    "-popularity": "-popularity",
+    "last_seen": "groupsearchviewlastvisited__last_visited",
```

Comment:
A bit of a nit but I'd prefer that the sort options more directly matched the model, so `visited` and `name` instead of `last_seen` and `alphabetical`